### PR TITLE
Source Trello: fixed `ixUpdate` board stream datatype

### LINKED ISSUE #21480: Source Trello: ixUpdate field has wrong datatype
## Environment
- **Airbyte version**:  v0.40.28
- **OS Version / Instance**: macOS

## Current Behavior
In trello connector, there is a field called ixUpated which has the wrong data type i.e ["null", "integer"] in board stream in discovery whereas from the backend API response, ixUpdated is returning as string.

## Expected Behavior
In discovery of board stream, ixUpated should be ["null", "string"]

## Are you willing to submit a PR?

Yes, I am submitting the PR for this issue.

### PR DESCRIPTION
**Pull request by zilehuda**
## What
FIxed #21480 

## How
- Changed the data type of `ixUpdate` from `["null", "integer"]` to `["null", "string"]`

### CODE REVIEW

File: airbyte-integrations/connectors/source-trello/source_trello/schemas/boards.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
       "type": ["null", "string"]
     },
     "ixUpdate": {
-      "type": ["null", "integer"]
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-trello/source_trello/schemas/boards.json:64]: From Trello documentation:
https://developer.atlassian.com/cloud/trello/rest/api-group-boards/#api-boards-id-get

image

image

Maybe you can show us an example with string from your dataset.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Please check my comment.

File: airbyte-integrations/connectors/source-trello/source_trello/schemas/boards.json (hunk 1 above)
Comment (zilehuda) [reply airbyte-integrations/connectors/source-trello/source_trello/schemas/boards.json:64]: @sajarin 
This endpoint is used in the stream: https://developer.atlassian.com/cloud/trello/rest/api-group-members/#api-members-id-boardstars-idstar-delete
reference: https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-trello/source_trello/source.py#L86
Seems like the Trello doc is also not upto-date.

Here you can see in the screenshot that ixUpdate is a string.

ixupdated-wrong

**Comment by sajarin** [comment]
Closing because this was apparently fixed in #24045

**Pull request closed without merging**